app/src/main/kotlin/org/wordpress/aztec/demo/MainActivity.kt

```diff
@@ -212,9 +221,11 @@ class MainActivity : AppCompatActivity(), OnRequestPermissionsResultCallback, Vi
 
         // initialize the text & HTML
         source.displayStyledAndFormattedHtml(EXAMPLE)
-        aztec.fromHtml(source.getPureHtml())
 
-        source.history = aztec.history
+        if (savedInstanceState == null) {
+            aztec.fromHtml(source.getPureHtml())
+            source.history = aztec.history
+        }
```

Comment:
This causes a crash if the history is not saved (i.e. `onSaveInstanceState`) and restored (i.e. `onRestoreInstanceState`) on device rotation.  Launch the app, rotate the device, tap the ***HTML*** format button, and the app crashes.  See the stack trace below.

```
02-14 09:41:46.935 7403-7403/org.wordpress.aztec E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.aztec, PID: 7403
    kotlin.UninitializedPropertyAccessException: lateinit property history has not been initialized
        at org.wordpress.aztec.source.SourceViewEditText.beforeTextChanged(SourceViewEditText.kt:121)
        at android.widget.TextView.sendBeforeTextChanged(TextView.java:8137)
        at android.widget.TextView.setText(TextView.java:4380)
        at android.widget.TextView.setText(TextView.java:4337)
        at android.widget.EditText.setText(EditText.java:89)
        at android.widget.TextView.setText(TextView.java:4312)
        at org.wordpress.aztec.source.SourceViewEditText.displayStyledAndFormattedHtml(SourceViewEditText.kt:166)
        at org.wordpress.aztec.toolbar.AztecToolbar.toggleEditorMode(AztecToolbar.kt:179)
        at org.wordpress.aztec.demo.MainActivity.onToolbarHtmlModeClicked(MainActivity.kt:547)
        at org.wordpress.aztec.toolbar.AztecToolbar.onToolbarAction(AztecToolbar.kt:170)
        at org.wordpress.aztec.toolbar.AztecToolbar.access$onToolbarAction(AztecToolbar.kt:18)
        at org.wordpress.aztec.toolbar.AztecToolbar$initView$1.onClick(AztecToolbar.kt:96)
        at android.view.View.performClick(View.java:5637)
        at android.widget.CompoundButton.performClick(CompoundButton.java:122)
        at android.view.View$PerformClick.run(View.java:22429)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.app.ActivityThread.main(ActivityThread.java:6119)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)
```